packages/terra-table/src/subcomponents/Row.jsx

```diff
@@ -99,6 +105,7 @@ function Row(props) {
       className={cx('row', {
         selected: isSelected,
         selectable: hasRowSelection,
+        'striped-table-row': isTableStriped,
```

Comment:
I think we could name this `striped`? This also feels awkward compared to the other class names that we use here.

Reply:
I think in that case, we can leave this as-is as was the intention that the table is the component that is passing this down to the component.